src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainViewStats.tsx

```diff
@@ -0,0 +1,217 @@
+          {exampleLinkText}
+          {showPointerBox && (
+            <PointerBox
+              anchor={anchorRef.current as HTMLDivElement}
+              onOutsideClick={() => setShowPointerBox(false)}
+              position={Position.BOTTOM_RIGHT}
+              container={containerRef.current}
+              autoAdjust={true}
+              classNames={{
+                box: styles.pointerBox,
+                pointer: styles.pointerBoxPointer
+              }}
+            >
+              <ViewInApp links={exampleLinks} />
+            </PointerBox>
+          )}
+        </span>
+      </span>
+    </div>
+  );
+};
+
+const getCollapsedContent = (statsSection: StatsSection) => {
+  const { primaryStats, secondaryStats, section, exampleLinks } = statsSection;
+  const { title, exampleLinkText } = sectionGroupsMap[section];
+
+  return (
+    <div className={styles.collapsedContent}>
+      <span className={styles.title}>{title}</span>
+      {primaryStats && (
+        <div className={styles.primary}>
+          <span className={styles.primaryValue}>
+            {primaryStats.primaryValue}
+          </span>
+          <span className={styles.primaryUnit}>{primaryStats.primaryUnit}</span>
+        </div>
+      )}
+
+      {secondaryStats && (
+        <div className={styles.secondary}>
+          <span className={styles.secondaryValue}>
+            {secondaryStats.secondaryValue}
+          </span>
+          <span className={styles.secondaryUnit}>
+            {secondaryStats.secondaryUnit}
+          </span>
+        </div>
+      )}
+
+      <ExampleLinks
+        exampleLinks={exampleLinks}
+        exampleLinkText={exampleLinkText}
+      />
+    </div>
+  );
+};
+
+const getExpandedContent = (statsSection: StatsSection) => {
+  const { groups, exampleLinks, section } = statsSection;
+  const { exampleLinkText } = sectionGroupsMap[section];
+  return (
+    <div className={styles.expandedContent}>
+      {groups.map((group, group_index: number) => {
+        const { title, stats } = group;
+        const primaryStatsClassName = classNames(styles.primaryStats, {
+          [styles.primaryStatsNoLink]: exampleLinks
+            ? !Object.keys(exampleLinks).length
+            : true
+        });
+        const primaryOrSecondaryClassName =
+          group_index === 0 ? primaryStatsClassName : styles.secondaryStats;
+        return (
+          <div key={group_index} className={primaryOrSecondaryClassName}>
+            <span className={styles.title}>{title}</span>
+            <div className={styles.stats}>
+              {stats.map((groupStats, group_index: number) => {
+                return (
+                  <div key={group_index}>
+                    {groupStats.map((stat, stat_index: number) => {
+                      return <SpeciesStats key={stat_index} {...stat} />;
+                    })}
+                  </div>
+                );
+              })}
+            </div>
+            {group_index === 0 && (
```

Comment:
I'd argue the way it's written now (`group_index === 0`) is ever so slightly more readable. With `!group_index` in this context you would have to mentally translate it back: "why is the group_index falsey? oh, that's because it's zero". The way it's written now saves this mental step.

Probably both ways of writing it are equally fine.

Reply:
If so then shouldn't we be using the same method in a single file as line 140 uses the other method

Reply:
🤷‍♂️